src/lib/contact/fullContact.ts

```diff
@@ -0,0 +1,34 @@
+import { accounts } from 'aleph-sdk-ts';
+import ContactFile from './contactClasses/fileContact';
+import Contact from './contact';
+import ContactFolder from './contactClasses/folderContact';
+import ManageContact from './contactClasses/manageContact';
+import Computing from './contactClasses/programContact';
+
+class FullContact {
+
+	public account: accounts.ethereum.ETHAccount;
```

Comment:
```suggestion
	public readonly account: accounts.ethereum.ETHAccount;
```
should add a bit more safety